src/Api/Controllers/EventsCreateController.php

```diff
@@ -23,7 +24,7 @@ protected function data(Request $request, Document $document)
         if(!$actor->can('event.create')){
             throw new PermissionDeniedException();
         }
-        $requestData = array_get($request->getParsedBody(), 'data.attributes');
+        $requestData = Arr::get($request->getParsedBody(), 'data.attributes');
```

Comment:
Just curious, why a library for native function replacement here?

Reply:
Part of laravel 6 upgrade : https://docs.flarum.org/extend/update-b14.html#removals